Content.Shared/Hands/Components/SharedHandsComponent.cs

```diff
@@ -44,27 +46,36 @@ public abstract class SharedHandsComponent : Component
     public float ThrowRange { get; set; } = 8f;
 }
 
+[DataDefinition]
 [Serializable, NetSerializable]
 public sealed class Hand //TODO: This should definitely be a struct - Jezi
 {
     [ViewVariables]
-    public string Name { get; }
+    [DataField("pssName")]
+    public string Name { get; private set; }
 
     [ViewVariables]
+    [DataField("pssLocation")]
     public HandLocation Location { get; }
 
     /// <summary>
     ///     The container used to hold the contents of this hand. Nullable because the client must get the containers via <see cref="ContainerManagerComponent"/>,
     ///     which may not be synced with the server when the client hands are created.
     /// </summary>
     [ViewVariables, NonSerialized]
+    [DataField("pssContainer")]
     public ContainerSlot? Container;
```

Comment:
afaik containers aren't serializable. you just need to store the container id, which I think can already be inferred from the hand name, and then get or create the container on startup.